src/MoBi.Presentation/MenusAndBars/ContextMenus/ContextMenuForSimulation.cs

```diff
@@ -50,8 +50,10 @@ public IContextMenu InitializeWith(IObjectBaseDTO dto, IPresenter presenter)
             createReportItemFor(simulation),
             createAddToJournal(simulation),
             exportSimulationResultsToExcel(simulation),
-            createExportForMatlabItem(simulation),
-            createMatlabODEExportItem(simulation),
+            createExportODEForMatlabItem(simulation),
+            createExportODEForRItem(simulation),
```

Comment:
I have added this one as developper only (not implemented yet but then it's there as soon as SimModel supports it)